tests/unittest.py

```diff
@@ -126,7 +127,8 @@ def tearDown(orig):
             ret = orig()
             # force a GC to workaround problems with deferreds leaking logcontexts when
             # they are GCed (see the logcontext docs)
-            gc.collect()
+            if platform.python_implementation() != "PyPy":
+                gc.collect()
```

Comment:
Does this call break or is it just slow? I'm a bit nervous that this will cause tests to not fail in some situations when they should?

Reply:
"to workaround problems with deferreds leaking logcontexts when they are GCed"

I recently heard a new twisted version will fix this problem, per @richvdh, but i'm not sure.

In the time i've tested this so far, nothing has broken, but i think we need a second opinion on this one, then.

Reply:
I guess my question was more of "what happens if you call `gc.collect()` on pypy?"

Reply:
It's fairly expensive, pypy has it's own GC implementation and the like, and looking at profile runs, it showed up fairly high (thus was pretty slow)

Reply:
Can we add that to the comment so that in the future we remember why we're skipping it?